script/Arcueid/Day 8/08_00_ARC08_6B.txt

```diff
@@ -402,7 +402,7 @@ If she were acting normally,
 [sha:d2cb585fa9da28a91e2d8b6323039dffa9081b4a]{
 -- Page 379, Offset 20621.
 -- ガレージの入り口から声をかける。
-I call out to her from the entrance to the garage.
+I call out to her from the entrance of the garage.
```

Comment:
Entrance `to` is more natural I feel. Ngrams seems to agree.

Reply:
I'm kinda indifferent here so i say revert if ngram agrees with ross